packages/i18next/src/lib/functions.ts

```diff
@@ -17,23 +32,28 @@ import type { ChannelTarget, InternationalizationContext, LocalizedMessageOption
  * @returns The name of the language key.
  */
 export function fetchLanguage(target: Target): Promise<string> {
+	// Handle Interactions:
+	if (target instanceof BaseCommandInteraction || target instanceof MessageComponentInteraction) {
+		return resolveLanguage({ channel: target.channel, guild: target.guild, user: target.user });
```

Comment:
I do approve of the rename to user from author now since it makes more sense in a way, but the fact that it is no longer ordered by scale is a little annoying. Can we ignore our instincts in terms of alphabetical ordering and put user at the start here?